contrib/mixin/mixin.libsonnet

```diff
@@ -238,6 +238,20 @@
               summary: 'etcd cluster database growing very fast.',
             },
           },
+          {
+            alert: 'etcdBackendHighFragmentationRatio',
+            expr: |||
+              (min_over_time(etcd_mvcc_db_total_size_in_use_in_bytes[5m]) / min_over_time(etcd_mvcc_db_total_size_in_bytes[5m])) < 0.5
+            ||| % $._config,
+            'for': '10m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              description: 'etcd cluster "{{ $labels.%s }}": database size in use on instance {{ $labels.instance }} is {{ $value | humanizePercentage }} of the actual allocated disk space, please run defragmentation procedure to retrieve the unused fragmented disk space.' % $._config.clusterLabel,
```

Comment:
Nit, we should use just `defragmentation` as it's a single command vs a procedure. I think we don't mention defragmentation as a procedure anywhere in the doc. So may be use something like `please run defragmentation (e.g. etcdctl defrag) to retrieve the...`). If we can provide link to defrag document as @ahrtr discussed in another comment that would be good otherwise the example using etcdctl I mentioned here should be good.